### PR TITLE
Check if script is run as root; Now hold and rmhold in one script. Completely redesigned script

### PR DESCRIPTION
**Pull request by jonisc**
Please try it before.

### CODE REVIEW

File: Gandalf-Portal/gandalf-hold [hunk 1]
```diff
@@ -1,36 +1,29 @@
...
-echo "Gandalf"
+echo "---Gandalf---"
 
 sleep 1
 
 echo "Created by EthanRDoesMC and mehulrao"
 
 sleep 1
-
+# Maybe better make it one script for all?
```
Comment (EthanRDoesMC) [inline Gandalf-Portal/gandalf-hold]: Perhaps. Like have the build.sh add in the version-specific stuff. Would certainly make things easier. Great idea!

Comment (jonisc) [inline Gandalf-Portal/gandalf-hold]: Look at /gandalf

**Comment by EthanRDoesMC** [comment]
Great stuff! Thanks for all of this! I literally don't know what else to say besides thank you.

**Comment by jonisc** [comment]
Thanks. Hopefully it'll compile correctly… only thing to do now: shrink app. Sorry for the question but the app is much too big.

**Comment by EthanRDoesMC** [comment]
i don't have the current source, so idk. That's mehulrao's little project.

**Comment by mehulrao** [comment]
I think it's because of the image files... Not sure tho

**Comment by mehulrao** [comment]
Gonna Merge

Looks Good!!

**Comment by jonisc** [comment]
Hmm. Maybe but the Frameworks folder is quite big too: 69,5 MB (Filza) I just created a small app in Xcode with only a bit of text and it was smaller than 1 MB. So what do you do with the app except showing the commands? I didn't need to add any Frameworks as I made the small app. The only "bad" thing about mine was

1. No icon
2. didn't look nice on every iPhone + iPad so I haven't included it and deleted it.

**Comment by mehulrao** [comment]
Yeah we figured that out yesterday. It's Swift

**Pull request merged**